config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -101,3 +108,140 @@ const (
 	// UserDefinedNamespaceIsolated enables monitoring for user-defined projects with namespace-scoped tenancy. This ensures that metrics, alerts, and monitoring data are isolated at the namespace level.
 	UserDefinedNamespaceIsolated UserDefinedMode = "NamespaceIsolated"
 )
+
+// alertmanagerMainConfig provides configuration options for the default Alertmanager instance
+// that runs in the `openshift-monitoring` namespace. Use this configuration to control
+// whether the default Alertmanager is deployed, how it logs, and how its pods are scheduled.
+//
+// Required: This field must be specified. a
+type AlertmanagerMainConfig struct {
+	// mode determines whether the default/main Alertmanager instance should be deployed
+	// as part of the monitoring stack. When set to "Enabled", the Cluster Monitoring Operator
+	// ensures that an Alertmanager instance is created and managed in the `openshift-monitoring` namespace.
+	// When set to "Disabled", the operator will not deploy the Alertmanager instance.
+	// Use this field if you want to explicitly opt in or out of running a platform-level Alertmanager.
+	// Required: This field must be specified.
+	// +kubebuilder:validation:Enum:=Enabled;Disabled;
+	// +required
+	DeploymentMode AlertManagerMode `json:"mode"`
+	// userMode controls whether Alertmanager should process configurations from user-defined (non-platform)
+	// namespaces for AlertmanagerConfig lookups.
+	// When set to true, Alertmanager will search for AlertmanagerConfig resources in user-defined namespaces.
+	// This field is only effective when the user workload Alertmanager instance is not enabled.
+	// If the user workload monitoring Alertmanager is enabled, this field is ignored.
+	// Required: This field must be specified.
+	// +kubebuilder:validation:Enum="";Enabled;Disabled
+	// +required
+	UserMode UserAlertManagerMode `json:"userMode"`
+	// logLevel defines the verbosity of logs emitted by Alertmanager.
+	// This field allows users to control the amount and severity of logs generated, which can be useful
+	// for debugging issues or reducing noise in production environments.
+	// Allowed values are:
+	// - `Error`: Logs only errors.
+	// - `Warn`: Logs warnings and errors.
+	// - `Info`: Logs general information, warnings, and errors. Default.
+	// - `Debug`: Logs detailed debug information.
+	// When omitted, the default value `Info` is used.
```

Comment:
Platform should choose the default here so we don't set the expectation that it will always and forever default to `Info`

Reply:
done